lib/timers.js

```diff
@@ -87,13 +87,20 @@ function listOnTimeout() {
 
   var first;
   while (first = L.peek(list)) {
-    // If the previous iteration caused a timer to be added,
```

Comment:
I am already using `Timer.now()` to do the computation and since we didn't need to update `now`, it's not used anywhere else and we quick return, I didn't think the previous comments made sense sticking around since we were no longer updating `now`.  Am I misunderstanding?